modin/core/io/text/csv_dispatcher.py

```diff
@@ -14,44 +14,38 @@
 """Module houses `CSVDispatcher` class, that is used for reading `.csv` files."""
 
 import pandas
+from pandas._typing import FilePathOrBuffer
 
 from modin.core.io.text.text_file_dispatcher import TextFileDispatcher
 
 
 class CSVDispatcher(TextFileDispatcher):
-    """
-    Class handles utils for reading `.csv` files.
+    """Class handles utils for reading `.csv` files."""
 
-    Inherits some common for text files util functions from `TextFileDispatcher` class.
-    """
+    callback = pandas.read_csv
 
     @classmethod
-    def _read(cls, filepath_or_buffer, **kwargs):
+    def check_parameters_support(
```

Comment:
There's no point of overriding parent's method if you change nothing. Just remove this method here 🙃